src/model/CollieConfig.ts

```diff
@@ -0,0 +1,49 @@
+import { Logger } from "../cli/Logger.ts";
+
+export class CollieConfig {
+  static CONFIG_FILE_PATH = ".collie/config.json";
+
+  static getFoundation(logger: Logger) {
```

Comment:
f: make this async (xSync calls are an anti-pattern and last-ditch escape, collie is built from the ground up to be async compatible)